pywwt/core.py

```diff
@@ -58,6 +58,16 @@ def _send_msg(self, **kwargs):
 
     # TODO: need to add more methods here.
 
+    def load_tour(self,url):
+        # should this throw an error if url doesn't end in '.wwt'?
```

Comment:
I don't think it makes sense to trust the extension only. But we should check what happens if the type of file is incorrect. Does the javascript print an error message?

Reply:
Currently there's no message. Nothing happens in the widget, either.

Reply:
Ok - having looked a bit at the docs I think we do want to check in here that the url ends in ``.wtt``. Could you add this?